components/userRequest.tsx

```diff
@@ -51,43 +58,49 @@ const UserRequestDashboardItem: React.FunctionComponent<UserRequest> = ({
       throw new Error(err.message);
     }
   };
+
+  // only show users emailVerified that are null
   return (
     <div>
-      <div className="grid grid-cols-4 gap-10 justify-items-start m-5">
-        <div className="flex flex-row">
-          <div className="w-10 h-10 mr-4 bg-placeholder rounded-full" />
-          <div className="w-32">
-            <p className="font-display self-center">{name}</p>
-            <p>User Role</p>
-          </div>
-        </div>
-        <div>
-          <p className="self-center font-normal">{email}</p>
-        </div>
-        <div>
-          <p className="self-center font-normal">{phoneNumber}</p>
-        </div>
-        <div>
-          <div className="flex space-x-4">
+      <div className="grid grid-cols-4 gap-2 justify-items-start m-5">
+        <Link href={`invite/userAccountPage/${id}`}>
+          <div className="flex space-x-40 bg-white hover:bg-placeholder">
+            <div className="flex flex-row">
+              <div className="w-10 h-10 mr-4 bg-placeholder rounded-full" />
+              <div className="w-32">
+                <p className="font-display self-center">{name}</p>
+                <p>User Role</p>
+              </div>
+            </div>
             <div>
-              <Button
-                className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded"
-                onClick={deleteUser}
-              >
-                Decline
-              </Button>
+              <p className="self-center font-normal">{email}</p>
             </div>
-            <div className="ml-4 ">
-              <div>
-                <Button
-                  className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded"
-                  onClick={acceptUser}
-                >
-                  Accept
-                </Button>
-              </div>
+            <div>
+              <p className="self-center font-normal">{phoneNumber}</p>
             </div>
           </div>
+        </Link>
+        <div className="flex space-x-35 position: absolute right-0">
+          {/* <div className="ml-40"> */}
+          <div>
+            <Button
+              className="bg-red-200 hover:bg-red-200 text-danger font-bold py-2 px-4 rounded"
+              onClick={deleteUser}
+            >
+              Decline
+            </Button>
+          </div>
+          {/* </div> */}
+          {/* <div className="ml-4"> */}
+          <div className="ml-4">
+            <Button
```

Comment:
The accept and delete buttons can be made into a component since you use them multiple times in both pages!